src/Application/Cli/PreviewCommand.php

```diff
@@ -90,6 +90,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
             sleep(1);
         }
 
+        $output->writeln('<error>HTTP server has stopped working</error>');
+
         return 0;
```

Comment:
I think we should since it's an error and every error must have an exit code greater than 0 and lesser than 254 (255 is PHP reserved).

The output is not silent, it shows all requests but when it fails depends on what happens, usually it just stops.

I think throwing an exception with the message you wrote is better. I will change this PR but you can request a change any time before merge it.